model_zoo/census_model_sqlflow/wide_and_deep/transform_ops.py

```diff
@@ -0,0 +1,74 @@
+from enum import Enum
+
+
+class TransformOpType(Enum):
+    HASH = (1,)
+    BUCKETIZE = (2,)
+    LOOKUP = (3,)
+    EMBEDDING = (4,)
+    CONCAT = (5,)
+    ARRAY = 6
+
+
+class TransformOp(object):
+    def __init__(self, name, input, output):
+        self.name = name
+        # The input name or a list of input names
+        self.input = input
+        # The output name
+        self.output = output
+        # The type of the TransformOp.
+        # The value is one of `TransformOpType`.
+        self.op_type = None
+
+
+class Vocabularize(TransformOp):
+    def __init__(
+        self, name, input, output, vocabulary_list=None, vocabulary_file=None
```

Comment:
We can merge `vocabulary_list` and `vocabulary_file` to a single argument `vocabulary`. Because the `IndexLookup` only has `vocabulary`.
https://github.com/sql-machine-learning/elasticdl/blob/402de98127621733ee2f1d8b2238a8ea681e0691/elasticdl_preprocessing/layers/index_lookup.py#L33-L35

Reply:
If we are using feature column, we may differentiate whether it's a vocabulary list or file.

Reply:
We can differentiate it by the type of `vocabulary`.

Reply:
Just like the information in other comments, the TransformOp in this python file is a sample describing the data structure which has the same fields in GO. It may be clearer if we differentiate whether it's a list of file name in the code generator component.